Rollbar.NetCore.AspNet/HttpRequestPackageDecorator.cs

```diff
@@ -84,6 +84,35 @@ protected override void Decorate(Data rollbarData)
             }
         }
 
+        /// <summary>
+        /// Assigns the request body.
+        /// </summary>
+        /// <param name="rollbarData">The rollbar data.</param>
+        private void AssignRequestBody(Data rollbarData)
+        {
+            string requestBodyString = GetBodyAsString(this._httpRequest);
+            object requesBodyObject = null;
```

Comment:
Not a big deal but this is a typo

Reply:
Opps! Thank you! Will correct with the next release...